index.html

```diff
@@ -10,32 +10,39 @@
     <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <div id="heading">
-      <header class="header-nav">
-        <a href="#" class="logo">MER<span>cy</span></a>
-        <nav id="hamburger">
-          <img src="/assets/images/bars.png" alt="menu" />
-        </nav>
-      </header>
-
-      <section class="heading">
-        <div class="header">
-          <h1>I’m MERCY <br />Glad to see <br />you!</h1>
+    <header class="landing-page">
+      <div class="header-nav">
+        <div class="logo">
+        <a href="#" class="logo-text">MER<span>cy</span></a>
         </div>
+        <nav class="menu">
+          <div class="bars">
+          <img id="hamburger" src="./assets/images/bars.png" alt="menu" />
+          </div>
+          <ul class="nav-items">
+            <li>
+              <a class="nav-link" href="#portfolio">Portfolio</a>
+            </li>
+            <li>
+              <a class="nav-link" href="#about">About</a>
+            </li>
+            <li>
+              <a class="nav-link" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
 
-        <div class="heading-paragraph">
-          <p>
+      <section class="headline">
```

Comment:
- [Optional] Consider reducing the space below the headline section to match the given template.
![image](https://user-images.githubusercontent.com/94551484/221346400-bd2adcbf-d3d1-4f18-a6c3-b68a1dd62b59.png)